apluslms_roman/configuration.py

```diff
@@ -22,30 +30,28 @@
     ),),)),
 ))
 
-class CourseConfigError(Exception):
+class RomanConfigError(Exception):
     pass
 
 def is_legacy_configuration(path):
     """Checks if path is legacy a-plus-rst-tools course"""
     return isfile(join(path, 'conf.py')) and isfile(join(path, 'Makefile'))
 
-class CourseConfig(Document):
-    name = 'course_config'
+class RomanConfig(Document):
```

Comment:
Actially, the class and the error class probably should ne `ProjectConfig` and `ProjectConfigError`. Furthermore, `RomanSettings` probably should be `GlobalSettings`, thus making these two classes clearly distinct.

Reply:
yes, that would make it a lot clearer